flow/core/kernel/kernel.py

```diff
@@ -53,10 +60,10 @@ def __init__(self, simulator):
         self.kernel_api = None
 
         if simulator == "traci":
-            self.simulation = None
-            self.scenario = None
-            self.vehicle = None
-            self.traffic_light = None
+            self.simulation = TraCISimulation(self)
+            self.scenario = KernelScenario(self)
+            self.vehicle = KernelVehicle(self, sim_params)
+            self.traffic_light = TraCITrafficLight(self)
```

Comment:
Add FIXME so we don't forget to make this non-traci as well

Reply:
don't worry, these are covered in later branched (which will be available to PR after this wave)